packages/forms/src/UIForm/fields/Code/Code.component.test.js

```diff
@@ -20,9 +20,14 @@ describe('Code field', () => {
 		value: 'toto',
 	};
 
+	afterEach(() => {
+		// clear JSDOM to avoid tests that mount on document.body to pollute other tests
+		document.getElementsByTagName('html')[0].innerHTML = '';
```

Comment:
```suggestion
		document.documentElement.innerHTML = '';
```

Reply:
You need to be carefull when using getElementsByTagName as it return an old HTMLCollection not a NodeList . It's not a problem here, just a reminder :)

Reply:
Thank you, I must admit that this time I'm guilty of copy/paste without really thinking, yeah that happens to everyone, don't judge me :D